salt/salt.spec

```diff
@@ -691,8 +691,12 @@ Group:          System/Management
 Requires:       %{name} = %{version}-%{release}
 Requires:       %{name}-master = %{version}-%{release}
 %if 0%{?default_py3}
+%if 0%{?suse_version}
 Requires:       python3-CherryPy >= 3.2.2
 %else
+Requires:       python3-cherrypy
+%endif
+%else
 Requires:       python-CherryPy >= 3.2.2
```

Comment:
I think we should have the same if-else for Python2. Could you add that please? Thanks!

Reply:
I have integrated the if statements in the top to avoid cluttering. The top if might also need some optimisation at some point.